tests/TestCase/View/XmlViewTest.php

```diff
@@ -253,13 +253,13 @@ public function testRenderWithSerializeTrue()
         $Request = new Request();
         $Response = new Response();
         $Controller = new Controller($Request, $Response);
-        $data = ['users' => ['user' => ['user1', 'user2']]];
+        $data = ['user' => ['user1', 'user2']];
         $Controller->set(['users' => $data, '_serialize' => true]);
         $Controller->viewClass = 'Xml';
         $View = $Controller->createView();
         $output = $View->render();
 
-        $this->assertSame(Xml::build($data)->asXML(), $output);
+        $this->assertSame(Xml::build(['response' => ['users' => $data]])->asXML(), $output);
```

Comment:
Ah OK. for JSON the change makes sense. However, XML serialization was fine previously (as long as your one var was not a scalar).